ee/models/event_definition.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+
+from posthog.models.event_definition import EventDefinition
+
+
+class EnterpriseEventDefinition(EventDefinition):
```

Comment:
@paolodamico I noticed that in the event list drawer we show `last modified by` time and user, that'll be additional fields to this correct? (updated_by, updated_at) ?

Reply:
Yup they would be additional, as they might differ from the owner. IIRC there's already models that have these attributes elsewhere in the code base.